### PR TITLE
Ports: Add lzop compression utility port

### PR DESCRIPTION
**Pull request by supercomputer7**

### CODE REVIEW

**Comment by circl-lastname** [comment]
Why not use a patch to remove the `malloc.h` include?

**Comment by supercomputer7** [comment]
> Why not use a patch to remove the `malloc.h` include?

Because it probably needs it to actually use `malloc` and I am not sure if it includes `stdlib.h` otherwise. I rather add a small header file in our LibC (which could be used for other ports as well in the future) than to put unnecessary patches on a port.
This just seems more correct to me, so I'd go with my way here.

**Comment by ADKaster** [comment]
`<malloc.h>` is a non-standard header. Looks like FreeBSD keeps it around and has a bunch of jemalloc stuff in it, while both OpenBSD (https://github.com/openbsd/src/commit/d88f57029e5acaaaf028633c7fa15c5d7325c5cc) and DragonflyBSD (https://github.com/DragonFlyBSD/DragonFlyBSD/commit/02b66c54cac986a0bf93435b8d5ae1b17521515b) have removed it.

I'd rather not add it, to be honest, and keep a patch for the application.

**Comment by supercomputer7** [comment]
> `<malloc.h>` is a non-standard header. Looks like FreeBSD keeps it around and has a bunch of jemalloc stuff in it, while both OpenBSD ([openbsd/src@d88f570](https://github.com/openbsd/src/commit/d88f57029e5acaaaf028633c7fa15c5d7325c5cc)) and DragonflyBSD ([DragonFlyBSD/DragonFlyBSD@02b66c5](https://github.com/DragonFlyBSD/DragonFlyBSD/commit/02b66c54cac986a0bf93435b8d5ae1b17521515b)) have removed it.
> 
> I'd rather not add it, to be honest, and keep a patch for the application.

I was not aware that this header is non-standard one. It's good to see that OpenBSD and DragonflyBSD removed that header. I added two small patches to ensure it doesn't try to include that header.
Thank you @ADKaster and @circl-lastname !

File: Ports/lzop/patches/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+## `0001-Don-t-try-to-use-malloc.h-header-file-in-the-codebas.patch`
+
+Don't try to use <malloc.h> header file in the codebase
+
+
+## `0002-Don-t-try-to-use-malloc.h-header-file-in-miniacc.h.patch`
+
+Don't try to use <malloc.h> header file in miniacc.h
+
```
Comment (timschumi) [inline Ports/lzop/patches/ReadMe.md:11]: Aren't these basically the same, with `miniacc.h` being a subset of "the codebase"? They are certainly fixing the same issue, so I'd put them into the same patch.

**Review by timschumi — CHANGES_REQUESTED** [review]

File: Ports/lzop/patches/ReadMe.md (hunk 1 above)
Comment (supercomputer7) [reply Ports/lzop/patches/ReadMe.md:11]: I see your point here, I'll merge these patches to one patch.

**Review by timschumi — APPROVED** [review]

**Pull request merged**